servers/camera/camera_feed.cpp

```diff
@@ -252,3 +263,29 @@ bool CameraFeed::activate_feed() {
 void CameraFeed::deactivate_feed() {
 	// nothing to do here
 }
+
+Array CameraFeed::get_formats() const {
+	Array result;
+	for (int i = 0; i < formats.size(); i++) {
```

Comment:
Id use `for each` syntax here, like so:
```suggestion
	for (const FeedFormat &format : formats) {
```